debian/control

```diff
@@ -7,6 +7,8 @@ Build-Depends: debhelper (>= 5),
                libcurl4-openssl-dev | libcurl4-gnutls-dev | libcurl3-openssl-dev | libcurl3-gnutls-dev,
                libglib2.0-dev (>= 2.30),
                libhiredis-dev,
+	       libevent-2.0-5,
```

Comment:
Since these are the build dependencies, it shouldn't be necessary to put the actual binary lib as dependency here. The -dev package should be all that's required (possibly with a >= 2.0 version restriction).